core/string/translation.cpp

```diff
@@ -1092,7 +1092,17 @@ Array TranslationServer::get_loaded_locales() const {
 		ERR_FAIL_COND_V(t.is_null(), Array());
 		String l = t->get_locale();
 
-		locales.push_back(l);
+		bool found = false;
+		for(int i = 0; i < locales.size(); i++){
+			if(locales[i] == l){
+				found = true;
+				break;
+     		}
+		}
+
+		if (!found) {
+			locales.push_back(l);
+		}
```

Comment:
`Array` has methods that can simplify this greatly, see `core/variant/variant.h`.

In particular, `has()` :)

Reply:
Also, it's not strictly needed to solve the referenced issue, but given that the order of translations can be arbitrary, the order of loaded locales can be too.

I think it would be worth changing `return locales;` to `return locales.sort();` to ensure that they're sorted alphabetically.

Reply:
Thanks for the feedback! I'll look into the methods of ```Array``` and also add the sort at the end and see if it all works as expected.

Reply:
@akien-mga I modified the code to use ```has()``` and it really made it a lot simpler! The only thing i wasn't able to do was ```return locales.sort()``` in one line, I had to split it in two for it to work.

This was the change without the sort applied(I jumbled the locales up so we could see the sort afterwards):
![WithoutSort](https://user-images.githubusercontent.com/26484801/145649316-83deb93a-5dca-4684-a04f-5955df3bd3fc.png)

And this was the change with the sort applied:
![WithSort](https://user-images.githubusercontent.com/26484801/145649400-68c41493-bfdb-448b-b7cc-75867a3e57ec.png)